### PR TITLE
Accept requests that are only stuck in Factory staging

### PR DESCRIPTION
**Pull request by lnussel**

### CODE REVIEW

File: check_source_in_factory.py [hunk 1]
```diff
@@ -148,7 +148,14 @@ def _check_requests(self, project, package, rev):
                         return True
                     elif req.state.name == 'review':
                         self.logger.info("request still in review")
-                        return None
+                        for r in req.reviews:
+                            if r.by_project and r.state == 'new' and r.by_project.startswith('openSUSE:Factory:Staging:'):
```
Comment (DimStar77) [inline check_source_in_factory.py]: the request will never be 'new' AND pending staging review - then it's still state == review

**Comment by coolo** [comment]
This is *heavily* undermotivated IMO.

**Comment by coolo** [comment]
and this is not breaking factory staging process ?

**Comment by lnussel** [comment]
do you have something in mind? AFAIK the bot is not used in Factory itself, only by maintenance and Leap

**Comment by coolo** [comment]
as long as it's handling this correctly, it might be fine

**Pull request merged**